README.md

```diff
@@ -316,3 +274,44 @@ necessary, and there are still known issues (see NetworkManager issue
 
 The Home Assistant Operating System 10 and newer correctly processes ICMPv6
 Router Advertisements.
+
+## Development
+
+Want to help out with development, testing, and/or documentation? Great! As both this project and Matter keeps evolving and more and more devices are available with actual Matter support, there will be a lot to improve. Reach out to us on discord if you want to help out.
+
+### Setting up your development environment
+
+**For enabling Matter support within Home Assistant, please refer to the Home Assistant documentation. These instructions are for development only!**
+
+Development is only possible on a (recent) Linux or MacOS machine. Other operating systems are not supported.
+
+- Download/clone the repo to your local machine.
+- Set-up the development environment: `scripts/setup.sh`
+
+You can check out the example script in the scripts folder for generic directions to run the client and server. To just run the server, you can run:
+
+```
+python -m matter_server.server
+
+Optional arguments:
+  -h, --help            show help message and exit
+  --vendorid VENDORID   Vendor ID for the Fabric, defaults to 65521
+  --fabricid FABRICID   Fabric ID for the Fabric, defaults to 1
+  --storage-path STORAGE_PATH
+                        Storage path to keep persistent data, defaults to $HOME/.matter_server
+  --port PORT           TCP Port to run the websocket server, defaults to 5580
+  --log-level LOG_LEVEL
+                        Provide logging level. Example --log-level debug, default=info, possible=(critical, error, warning, info, debug)
+  --log-file LOG_FILE   Log file to write to (optional).
```

Comment:
These are quite outdated, maybe we should remove them from here anyways?

Reply:
yeah, good idea

Reply:
adjusted